libs/ui-utils/src/lib/toPixelValue.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Converts a number to a pixel string, or returns the string value as-is
+ * @param value - Number (converted to px) or string (returned as-is)
+ * @returns CSS value string or undefined
+ * @example
+ * toPixelValue(100) // '100px'
+ * toPixelValue('50%') // '50%'
+ * toPixelValue(undefined) // undefined
+ */
+export function toPixelValue(value?: number | string): string | undefined {
```

Comment:
There is a difference between `optional` and `undefined` types:
```

function toPixelValue(value?: number | string): string

const result = toPixelValue() // all good, code compiles

function toPixelValue(value: number | string | undefined): string

const result = toPixelValue() // code doesn't compile, value must be specified

```

Since we always expect a value passed to the function, it should has `undefined` type instead of optional

Reply:
Addressed